blocks/library/freeform/old-editor.js

```diff
@@ -67,6 +70,16 @@ export default class OldEditor extends Component {
 			} );
 		} );
 
+		editor.on( 'keydown', ( event ) => {
+			if ( ( event.keyCode === BACKSPACE || event.keyCode === DELETE ) &&
+					/^\n?$/.test( editor.getContent( { format: 'text' } ) ) ) {
```

Comment:
Though it does fire the 'GetContent' event by default. I could turn that off by passing ```{ format: 'text', no_events: true }``` .